rosdep/python.yaml

```diff
@@ -3673,6 +3687,7 @@ python-sympy:
   fedora: [sympy]
   gentoo: [dev-python/sympy]
   ubuntu:
+    bionic: [python-sympy]
```

Comment:
Can you please also add artful and cosmic ?

Reply:
Done !